Samples/MyTestMacOSApp/MyTestMacOSApp/ADTestAppCache.m

```diff
@@ -141,7 +145,7 @@ - (OSStatus)readFromKeychain
         CFDataRef data = NULL;
         OSStatus status = SecItemCopyMatching((CFDictionaryRef)readQuery, (CFTypeRef *)&data);
         
-        _data = (__bridge NSData*)data;
+        _data = (__bridge_transfer NSData*)data;
```

Comment:
An interesting discussion here: http://stackoverflow.com/questions/16780202/secitemcopymatching-still-leak-on-osx-under-arc

If I am understanding correctly, the readQuery can also leak if not disposed.